pkg/abaputils/abaputils_test.go

```diff
@@ -44,10 +46,11 @@ func TestCloudFoundryGetAbapCommunicationInfo(t *testing.T) {
 		assert.Equal(t, "", connectionDetails.XCsrfToken)
 
 		assert.EqualError(t, err, "Parameters missing. Please provide EITHER the Host of the ABAP server OR the Cloud Foundry ApiEndpoint, Organization, Space, Service Instance and a corresponding Service Key for the Communication Scenario SAP_COM_0510")
-		assert.Error(t, err)
 	})
 	t.Run("CF GetAbapCommunicationArrangementInfo - Error - reading service Key", func(t *testing.T) {
 
+		expectedErrorMessage := "Read service key failed: Parsing the service key failed. Service key is empty"
```

Comment:
[minor] also that could be inlined. I understand that having a variable `expectedErrorMessage` makes it very clear that this message is expected. But would that not also be clear when having this directly in the `assert.EqualError(./.)